src/main/java/com/spotify/github/v3/issues/Issue.java

```diff
@@ -91,8 +91,9 @@
   @Nullable
   List<Label> labels();
 
-  /** Login for the user that this issue should be assigned to. */
-  Optional<User> assignee();
```

Comment:
I see that [prs/PullRequestItem](https://github.com/spotify/github-java-client/blob/master/src/main/java/com/spotify/github/v3/prs/PullRequestItem.java#L87-L91) has both `Optional<User> assignee()` and `Optional<List<User>> assignees()`, should this be kept for backwards compatibility?